service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.workspace.common.fixtures;
+
+import bio.terra.workspace.db.WorkspaceDao;
+import bio.terra.workspace.service.workspace.model.CloudPlatform;
+import bio.terra.workspace.service.workspace.model.GcpCloudContext;
+import java.util.UUID;
+
+public class WorkspaceFixtures {
+
+  public static void createGcpCloudContext(
```

Comment:
nit: add javadoc. it's not clear from the name that this is a direct DAO operation that skips the flight (and picks a fake flight ID), it would be good to document that